README.md

```diff
@@ -288,86 +270,51 @@ Contributions go far beyond pull requests and commits. Although we love giving y
+- [The Contributor's Guide to webpack](https://medium.com/webpack/contributors-guide/home)
 
 To get started have a look at our [documentation on contributing](https://github.com/webpack/webpack/blob/main/CONTRIBUTING.md).
 
-If you are worried or don't know where to start, you can **always** reach out to [Sean Larkin (@TheLarkInn) on Twitter](https://twitter.com/thelarkinn) or simply submit an issue and a maintainer can help give you guidance!
-
-We have also started a series on our [Medium Publication](https://medium.com/webpack) called [The Contributor's Guide to webpack](https://medium.com/webpack/contributors-guide/home). We welcome you to read it and post any questions or responses if you still need help.
-
-_Looking to speak about webpack?_ We'd **love** to review your talk abstract/CFP! You can email it to webpack [at] opencollective [dot] com and we can give pointers or tips!!!
-
-<h3 align="center">Creating your own plugins and loaders</h3>
+<h3>Creating your own plugins and loaders</h3>
 
 If you create a loader or plugin, we would <3 for you to open source it, and put it on npm. We follow the `x-loader`, `x-webpack-plugin` naming convention.
 
-<h2 align="center">Support</h2>
+<h2>Support</h2>
 
 We consider webpack to be a low-level tool used not only individually but also layered beneath other awesome tools. Because of its flexibility, webpack isn't always the _easiest_ entry-level solution, however we do believe it is the most powerful. That said, we're always looking for ways to improve and simplify the tool without compromising functionality. If you have any ideas on ways to accomplish this, we're all ears!
 
 If you're just getting started, take a look at [our new docs and concepts page](https://webpack.js.org/concepts/). This has a high level overview that is great for beginners!!
 
-Looking for webpack 1 docs? Please check out the old [wiki](https://github.com/webpack/docs/wiki/contents), but note that this deprecated version is no longer supported.
+If you have discovered a 🐜 or have a feature suggestion, feel free to create an issue on GitHub.
 
-If you want to discuss something or just need help, [here is our Gitter room](https://gitter.im/webpack/webpack) where there are always individuals looking to help out!
+<h2>Current project members</h2>
 
-If you are still having difficulty, we would love for you to post
-a question to [StackOverflow with the webpack tag](https://stackoverflow.com/tags/webpack). It is much easier to answer questions that include your webpack.config.js and relevant files! So if you can provide them, we'd be extremely grateful (and more likely to help you find the answer!)
+For information about the governance of the Node.js project, see [GOVERNANCE.md](./GOVERNANCE.md).
 
-If you are twitter savvy you can tweet #webpack with your question and someone should be able to reach out and help also.
+<h3>TSC (Technical Steering Committee)</h3>
```

Comment:
@evenstensberg can you confirm this list is accurate and represents the current TSC?

Reply:
Emanuelle is out, other then that it looks ok